### PR TITLE
feature(charts/dex): added priority class name support

### PR DESCRIPTION
**Pull request by christianbeland**

### CODE REVIEW

File: charts/dex/Chart.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ maintainers:
 annotations:
   artifacthub.io/changes: |
     - Fix missing HTTPS service values
```
Comment (sagikazarmark) [inline charts/dex/Chart.yaml:24]:
```suggestion
```

File: charts/dex/templates/deployment.yaml [hunk 2]
```diff
@@ -21,6 +21,9 @@ spec:
       labels:
         {{- include "dex.selectorLabels" . | nindent 8 }}
     spec:
+      {{- if .Values.priorityClassName }}
+      priorityClassName: {{ .Values.priorityClassName | quote }}
+      {{- end }}
```
Comment (sagikazarmark) [inline charts/dex/templates/deployment.yaml:26]:
```suggestion
      {{- with .Values.priorityClassName }}
      priorityClassName: {{ . | quote }}
      {{- end }}
```

File: charts/dex/values.yaml [hunk 3]
```diff
@@ -24,6 +24,9 @@ nameOverride: ""
 # -- A name to substitute for the full names of resources.
 fullnameOverride: ""
 
+## Set pod priorityClassName
```
Comment (sagikazarmark) [inline charts/dex/values.yaml:27]:
```suggestion
# -- Specify a priority class name to set [pod priority](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#pod-priority).
```

File: charts/dex/values.yaml [hunk 4]
```diff
@@ -24,6 +24,9 @@ nameOverride: ""
...
+# priorityClassName: ""
```
Comment (sagikazarmark) [inline charts/dex/values.yaml:28]:
```suggestion
priorityClassName: ""
```

**Review by sagikazarmark — CHANGES_REQUESTED** [review]
Thanks @christianbeland !

Can you please take a look at the requested changes?

**Comment by christianbeland** [comment]
Accepted all changes.  thanks

**Pull request closed without merging**